packages/fx-core/src/plugins/resource/localdebug/launch.ts

```diff
@@ -27,6 +27,7 @@ export function generateConfigurations(
         group: "remote",
         order: edgeOrder,
       },
+      timeout: 20000,
```

Comment:
Could this timeout be set via debug provider?

Reply:
Yes, just like resolving the url in debug provider. But if so, user will not modify the timeout if it is not enough.

Reply:
Emm, so is it possible that:
- if user defines timeout in tasks.json, do nothing
- else, add `timeout: 20000` as default value in debug provider

This will affect all existing projects without any user-side modification.

Reply:
Nice idea. I did not consider the existing projects. Will update it later.